test/api/v3/unit/models/group.test.js

```diff
@@ -460,73 +457,67 @@ describe('Group Model', () => {
         };
       });
 
-      it('throws an error if no uuids or emails are passed in', (done) => {
-        try {
-          Group.validateInvitations(null, null, res);
-        } catch (err) {
-          expect(err).to.be.an.instanceof(BadRequest);
-          expect(res.t).to.be.calledOnce;
-          expect(res.t).to.be.calledWith('canOnlyInviteEmailUuid');
-          done();
-        }
+      it('throws an error if no uuids or emails are passed in', async () => {
+        await expect(Group.validateInvitations(null, null, res)).to.eventually.be.rejected.and.eql({
+          httpCode: 400,
+          message: 'Bad request.',
```

Comment:
Yes, I'm wondering why the message is `Bad request` and not the real reason, it should not be `Bad message`. Does the test fails if `message: 'Bad request.'` becomes `message: shared.i18n.t('canOnlyInviteEmailUuid')`? 

I can't see where `Bad request.` as a message would come from. It should be like this https://github.com/HabitRPG/habitica/pull/8589/files#diff-ee4d06fc0df454c1e75431b5abb56e39R455